### PR TITLE
Add CHM portal entrance

### PR DESCRIPTION
**Pull request by liyuanmontreal**

### CODE REVIEW

File: app/templates/chm/index.ejs [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+      <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>
+    <![endif]-->
+
+</head>
+<body ng-controller="ChmTemplateController" id="top" lang="<%=preferredLanguage||'en'%>">
+    <div id="g-recaptcha"></div>
+    <span style='display:none' id="appTitle">Biosafety Clearing-House</span>
```
Comment (blaisefonseca) [inline app/templates/chm/index.ejs:41]: Clearing-House Mechanism

**Review by blaisefonseca — COMMENTED** [review]
@liyuanmontreal Nice work on you first PR, please just replace Biosafety Clearing-House with Clearing-House Mechanism

File: app/templates/chm/index.ejs (hunk 1 above)
Comment (liyuanmontreal) [reply app/templates/chm/index.ejs]: done

File: app/templates/chm/index.ejs [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    <span style='display:none' id="appDescription">The Biosafety Clearing-House</span>
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/templates/chm/index.ejs]: done

File: app/templates/chm/index.ejs [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+            <div class="animated-container">
+                <div class="messaging">
+                    <span class="color-orange ng-scope" ng-if="development_env" style="font-size:48px;font-weight:bold;"> 
+                        <img src="/app/img/cbd-logo-green-en.svg" style="height:80px;">
+                    </span>
+                   
+                    <h1>
+                        <br/>                  
+                       <div class="color-black" id="appLoadingMessage">
+                            loading the <br/>Biosafety Clearing-House
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/templates/chm/index.ejs]: done

**Comment by liyuanmontreal** [comment]
Finish editing. Please review again, thanks!

**Review by blaisefonseca — COMMENTED** [review]
please also make a copy of the realm-configuration in the project
https://api.cbd.int/api/v2018/realm-configurations/chm.cbd.int

**Pull request merged**